ibis/__init__.py

```diff
@@ -43,6 +39,11 @@ def __getattr__(name: str) -> BaseBackend:
     the `ibis.backends` entrypoints. If successful, the `ibis.sqlite`
     attribute is "cached", so this function is only called the first time.
     """
+    # Converting an Ibis schema to a pandas DataFrame requires registering some
+    # type conversions that are currently registered in the pandas backend
+    import ibis.backends.pandas
+    from ibis import util
```

Comment:
Based on the change in the `__all__` variable, I think that `ibis.backends.pandas` wasn't public but `ir` was.

Reply:
Yep, `ir` is not something we want to be public long term so I removed it here.

Reply:
I think I can break that out into a separate PR so that it'll get listed as a breaking change.